### PR TITLE
Create 4||6 content card deck support

### PR DESCRIPTION
**Pull request by B77Mills**
update content card deck to be more configurable

Configurations:
 - cols
 - display-ads
 - with-teaser

This should support 4 & 6 column display in one row.

Screen Shot 2020-01-02 at 11 49 55 AM

### CODE REVIEW

File: sites/firehouse.com/server/styles/index.scss [hunk 1]
```diff
@@ -41,3 +41,45 @@ $body-bg: #fff;
...
 }
+
+// Override to make card deck flows with 4 cols display 25% till it reaches a lower resolution
+@media (min-width: 768px) {
+  .card-deck-flow {
+    $self: &;
+    &--4-cols {
+      #{ $self }{
+        &__node {
+          flex: 0 0 25%;
```
Comment (zarathustra323) [inline sites/firehouse.com/server/styles/index.scss]: I believe you can use the media breakpoint and col Mixins to handle this

Comment (zarathustra323) [reply sites/firehouse.com/server/styles/index.scss]: See https://getbootstrap.com/docs/4.4/layout/overview/

And the bottom of this page:

https://getbootstrap.com/docs/4.4/layout/grid/

Comment (zarathustra323) [reply sites/firehouse.com/server/styles/index.scss]: If those don’t work let me know and I’ll merge as-is

Comment (B77Mills) [reply sites/firehouse.com/server/styles/index.scss]: Worked Great!

**Pull request merged**